server/lib/nl/fulfillment/fulfiller.py

```diff
@@ -161,7 +164,12 @@ def _perform_strict_mode_checks(uttr: Utterance) -> bool:
 
 
 def _produce_query_types(uttr: Utterance) -> List[QueryType]:
-  query_types = [handlers.first_query_type(uttr)]
+  query_types = []
+  if params.is_bio(uttr.insight_ctx) and uttr.entities:
+    query_types.append(QueryType.PROPVALS)
+    if not uttr.places:
```

Comment:
We fall-through if there are places?  Do we then support fallback from PROPVALS to another entity type?